feature/setting/src/main/java/io/github/droidkaigi/confsched2022/feature/setting/Setting.kt

```diff
@@ -64,30 +73,17 @@ fun Setting(
             verticalArrangement = Arrangement.Top,
             horizontalAlignment = Alignment.Start
         ) {
-            SettingItem.values().forEach { item ->
-                SettingItem(
-                    icon = item.icon,
-                    title = stringResource(item.titleStringRes)
-                )
-            }
+            DarkModeSetting()
+            LanguageSetting()
         }
     }
 }
 
-enum class SettingItem(val titleStringRes: StringResource, val icon: ImageVector) {
-    DarkMode(Strings.setting_item_dark_mode, Icons.Default.DarkMode),
-    Language(Strings.setting_item_language, Icons.Default.Language)
-}
-
 @Composable
-fun SettingItem(
-    icon: ImageVector,
-    title: String,
-    modifier: Modifier = Modifier,
-) {
+fun DarkModeSetting() {
```

Comment:
Sorry, can you delete this setting? This is last year's app copy and this is not needed 🙏

Reply:
Sure! 🙋